tests/robot-tests/scripts/create_snapshots.py

```diff
@@ -0,0 +1,145 @@
+import os
```

Comment:
Would you be against putting a quick description of what this script does at the top here?  Or would you prefer to document it in another way?